libs/client/cornflow_client/core/experiment.py

```diff
@@ -227,4 +228,4 @@ def get_solver_config(
                     }
                 )
 
-        return conf
+        return solver, conf
```

Comment:
I think this will break a lot of project which are using get_solver_config.
It would be better to create a get_solver method in execution and use it in this function